manager/manager.go

```diff
@@ -1103,7 +1133,9 @@ func defaultClusterObject(
 	encryptionConfig api.EncryptionConfig,
 	initialUnlockKeys []*api.EncryptionKey,
 	rootCA *ca.RootCA,
-	fips bool) *api.Cluster {
+	fips bool,
+	defaultAddressPool []string,
```

Comment:
if this returns a default config, why do you pass these?? it should auto populate with default values

Reply:
@fcrisciani 's point seems valid. The other params in the defaultClusterObject() function are defaults, whereas the new arguments are  not code defaults. Maybe add a new function which assigns the new arguments to the cluster object.

Reply:
when we create default cluster we have hard coded values and also we get user input value.  I felt its redundant work to call again cluster.update in the same code path even though we know the the values when we create default cluster object. 
Anyways I will change it.

Reply:
@anshulpundir @thaJeztah  updated as per your suggestion

Reply:
didn't see this update. Is a new patch forthcoming? @selansen

Reply:
I modified the code. now default cluster constructor calls with nil value.  I used update culster to update it once default cluster object get created.